RELEASE-NOTES.md

```diff
@@ -1,5 +1,65 @@
 # Prefect Release Notes
 
+## Release 2.13.6
+
+### Specify a default result storage block as a setting
+
+Previously, specifying result storage blocks necessitated changes in the `@flow` or `@task` decorator. Now, the `PREFECT_DEFAULT_RESULT_STORAGE_BLOCK` setting allows users to set a default storage block on a work pool or via job variables for a deployment. For example, to set a default storage block for a deployment via `prefect.yaml`:
+
+```yaml
+- name: my-super-cool-deployment
+  entrypoint: some_directory/some_file.py:my_flow
+  schedule:
+    cron: "0 20 * * 1-5"
+  work_pool:
+    name: ecs-pool
+    job_variables:
+      env:
+        PREFECT_DEFAULT_RESULT_STORAGE_BLOCK: s3/my-s3-bucket-block-name
+```
```

Comment:
should we specify this as being in the deployment section?

Reply:
yep - good catch